README.md

```diff
@@ -56,3 +55,140 @@ service, some of these values may be required to set.
+
+#### Bucket creation
+Create a new bucket in the AWS S3 console with the settings below:
+
+1. `Bucket name`: openedx-profile-pictures-bucket
+2. `AWS Region`: no need to change
+3. `Object Ownership`: ACLs disabled (recommended)
+4. `Block Public Access settings for this bucket`: Uncheck "Block all public access"
+5. `Bucket Versioning`: Disable
+6. `Tags`: It is optional
+7. `Default encryption`: Disable
+8. Then click: **Create bucket**
+
+#### Bucket Permissions
+We need to configure the bucket to allow Open edX to access (`GET`) the files using a simple URL but only from the domain your Open edX instance is hosted on. Also give full access (read, write, delete, list) to the backend user account (authenticated with `OPENEDX_AWS_ACCESS_KEY` and `OPENEDX_AWS_SECRET_ACCESS_KEY`).
+
+Access the Bucket Permissions tab and change the configurations below:
+
+**Bucket policy**:
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "openedxWebAccess",
+            "Effect": "Allow",
+            "Principal": "*",
+            "Action": "s3:GetObject*",
+            "Resource": "arn:aws:s3:::${AWS_PROFILE_BUCKET_NAME}/*"
+        },
+        {
+            "Sid": "openedxBackendAccess",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::${AWS_ACCOUNT_ID}:user/${AWS_IAM_USER}"
+            },
+            "Action": "s3:*",
+            "Resource": "arn:aws:s3:::${AWS_PROFILE_BUCKET_NAME}/*"
+        }
+    ]
+}
+```
+**Note**: Make sure you replace `AWS_PROFILE_BUCKET_NAME`, `AWS_ACCOUNT_ID` and `AWS_IAM_USER` with correct values.
+
+**Cross-origin resource sharing** (CORS):
+```
+[
+    {
+        "AllowedHeaders": [],
+        "AllowedMethods": [
+            "GET"
+        ],
+        "AllowedOrigins": [
+            "https://${LMS_HOST}",
+            "https://${CMS_HOST}"
+        ],
+        "ExposeHeaders": [],
+        "MaxAgeSeconds": 3000
+    }
+]
+```
+
+**Note**: Replace `LMS_HOST` and `CMS_HOST` with URLs of your LMS and CMS.
+
+### Examples of variable values:
+* `LMS_HOST`: openedx.com
+* `CMS_HOST`: studio.openedx.com
+* `AWS_PROFILE_BUCKET_NAME`: openedx-profile-pictures-bucket
+* `AWS_ACCOUNT_ID`: 123456789012
+* `AWS_IAM_USER`: openedx-user
+
+## CloudFront
+If you need to accelerate website content delivery, another option to serve the profile pictures is to use [CloudFront](https://aws.amazon.com/cloudfront/) in front of AWS S3.
+
+Amazon CloudFront is a fast content delivery network ([CDN](https://en.wikipedia.org/wiki/Content_delivery_network)) service that securely delivers data to customers globally with low latency and high transfer speeds.
+
+CloudFront is set up in which data is cached and will be sent to the user from that nearest edge location instead of picking it from the original location (S3 Bucket) thus speeding up the transfer rate.
```

Comment:
```suggestion
CloudFront caches the data and sends it to the user from the nearest edge location, instead of picking it from the original location (S3 Bucket). This speeds up the transfer rate.
```
Break the long sentence. Also using the active voice is recommended.